src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/MergeDataToSearchResultOperation.java

```diff
@@ -17,37 +17,43 @@
 @Named
 @Singleton
 public class MergeDataToSearchResultOperation {
+    public static final String DRAFT_ID = "DRAFT%s";
     private final UIDefinitionRepository uiDefinitionRepository;
 
     public MergeDataToSearchResultOperation(final UIDefinitionRepository uiDefinitionRepository) {
         this.uiDefinitionRepository = uiDefinitionRepository;
     }
 
-    public SearchResultView execute(final CaseType caseType, final List<CaseDetails> caseDetails,final String view) {
+    public SearchResultView execute(final CaseType caseType, final List<CaseDetails> caseDetails, final String view, final String resultError) {
         final SearchResult searchResult = getSearchResult(caseType, view);
         final SearchResultViewColumn[] viewColumns = Arrays.stream(searchResult.getFields())
             .flatMap(searchResultField -> caseType.getCaseFields().stream()
-            .filter(caseField -> caseField.getId().equals(searchResultField.getCaseFieldId()))
-            .map(caseField ->  new SearchResultViewColumn(
-                searchResultField.getCaseFieldId(),
-                caseField.getFieldType(),
-                searchResultField.getLabel(),
-                searchResultField.getDisplayOrder())
-             ))
+                .filter(caseField -> caseField.getId().equals(searchResultField.getCaseFieldId()))
```

Comment:
this sequence of flatmap filtler map it's a bit difficult to follow. Can we break it up in and give the intermediate results meaningful names?